app/services/k8s/usage-metrics.ts

```diff
@@ -84,28 +127,32 @@ export async function getPodMetrics(
     // Map over containers to collect their usage, limits, and requests
     const containers = item.containers
       .filter((container) => {
-        if (container.name === 'POD' && container.usage.cpu === '0' && container.usage.memory === '0') return false;
+        if (container.name === 'POD') return false;
         return true;
       })
       .map((container, index) => {
         const resourceDef = podStatus.body.spec?.containers[index]?.resources ?? {};
```

Comment:
is it reliable to find the container by index, if the container has name, is it better to use the name to fine it? if then, is the name always unique?

Reply:
When querying pod metrics, Kubernetes adds a pseudo-container named `POD` to represent the pod's management overhead, like network or logging. This is not an actual application container and usually shows zero resource usage `(cpu: '0', memory: '0')`. looks like usually it append as the last array item, but not sure if it is a good idea to relay on `last index`